src/components/core/tooltip/tooltip/stories/tooltip.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import CheckIconSVG from '@assets/svg/circular-check-icon.svg';
+import Tooltip, { TooltipStory } from '../tooltip';
+
+export default {
+    title: 'Tooltip / Tooltip',
+    component: Tooltip,
+    argTypes: {
+        type: {
+            description: 'controls the type of tooltip ',
+            control: {
+                type: 'select',
+                options: ['default', 'error'],
+            },
+            defaultValue: 'error',
+        },
+        tooltip_content: {
+            defaultValue: <span>this is a tooltip</span>,
+        },
+        side: {
+            description: 'controls the side of tooltip ',
+            control: {
+                type: 'select',
+                options: ['top', 'bottom', 'left', 'right'],
+            },
+            defaultValue: 'top',
+        },
+        is_fixed_width: {
+            description: 'If set to `true`, width will be fixed.',
+            table: {
+                type: { summary: 'boolean' },
+                defaultValue: { summary: true },
+            },
+        },
+    },
+} as ComponentMeta<typeof TooltipStory>;
+
+const Template: ComponentStory<typeof TooltipStory> = (args) => (
```

Comment:
Can we provide a centered container for this?
right now the content is not visible in the story book :pray: 
The component is working correctly but for the sake of testing and visual testing we kinda have to show the component in a centered container.